devtools/platform-descriptor-json/src/main/resources/bundled-codestarts/buildtool/gradle/base/..gitignore

```diff
@@ -0,0 +1,3 @@
+# Gradle
```

Comment:
Is it called `..gitignore`? With two dots at the beginning?

Reply:
haha good catch :) I wasn't sure someone would notice. Without this hack, the `.gitignore` is interpreted by git and also even worst excluded from the jar. This is only required if it's a static file, no need for it when it uses qute: `.gitignore.tpl.qute`:
https://github.com/ia3andy/quarkus/blob/codestarts-v0/independent-projects/tools/codestarts/src/main/java/io/quarkus/devtools/codestarts/reader/CodestartFileReader.java#L36